test/test.js

```diff
@@ -10,8 +13,31 @@ jasmine.loadConfig({
     ]
 });
 
-var runner =  new JsReporters.JasmineAdapter(jasmine);
-var testReporter = new JsReporters.TestReporter(runner, referenceData.Jasmine);
+var jasmineRunner = new JsReporters.JasmineAdapter(jasmine.env);
+jasmine.addReporter({}); // Suppress the default reporter
+var jasmineTestReporter = new JsReporters.TestReporter(jasmineRunner, referenceData.Jasmine);
 
 jasmine.execute();
-process.exit(testReporter.ok ? 0 : 1);
+if (!jasmineTestReporter.ok) {
+    throw new Error("Tests for JasmineAdapter failed!");
+}
+
+//TODO: Use a proper test framework?
```

Comment:
Now I got it. Still don't understand how an unhandled exception doesn't lead to a non zero exit code. How does the assert module work then?Afaik that can be used by itself without any extra infrastructure. 

@leobalter any idea?